site/docs/advanced/flood.md

```diff
@@ -5,29 +5,119 @@ next: ./transformers.md
-It is an [API transformer function](./transformers.md) that does exactly that.
+That plugin is [very simple](https://github.com/grammyjs/auto-retry/blob/main/src/index.ts).
+It literally just sleeps and retries.
+However, using it has a major implication: **any request can be slow**.
+This means that when you run your bot on webhooks, [you technically have to use a queue](../guide/deployment-types.md#ending-webhook-requests-in-time) no matter what you do, or else you need to configure the auto-retry plugin in a way that it never takes a lot of time---but then your bot may skip some requests.
 
-However, if the traffic to your bot increases rapidly, e.g. when it is added to a large group, it may run into a lot of rate limiting errors before the traffic spike settles.
-This could lead to a ban.
-Moreover, as requests might be tried several times, your server will consume more RAM and bandwidth than necessary.
-Instead of fixing the problem after the fact, it is much better to enqueue all API requests and only send them at the permitted speed:
+## What the Exact Limits Are
 
-## The Real Solution (recommended)
+They are unspecified.
 
-grammY provides you with the [throttler plugin](../plugins/transformer-throttler.md) that automatically makes your bot respect all rate limits by enqueuing the outgoing requests of your bot in a message queue.
-This plugin is just as simple to set up but does a much better job at flood control.
-There isn't really any good reason to use [auto-retry](../plugins/auto-retry.md) over the [throttler plugin](../plugins/transformer-throttler.md).
-In some cases it may make sense to use both.
+Deal with it.
+
+We have some good ideas about how many requests you can perform, but the exact numbers are unknown.
+(If someone tells you the limits, they are either not very well informed, or lying.)
+The limits are not simply hard thresholds that you can find out by experimenting with the Bot API.
+Rather, they are flexible constraints that change based on your bot's exact request payloads, the number of users, and other factors, not all of which are known.
+
+Here are a few misconceptions and false assumptions about rate limits.
+
+- My bot is too new to receive flood wait errors.
+- My bot does not get enough traffic to receive flood wait errors.
+- This feature of my bot is not used enough to receive flood wait errors.
+- My bot leaves enough time between API calls so it will not receive flood wait errors.
+- This particular method call cannot receive flood wait errors.
+- `getMe` cannot receive flood wait errors.
+- `getUpdates` cannot receive flood wait errors.
+
+All of these things are wrong.
+
+Let's get to the things we _do_ know.
+
+## Safe Assumptions About Rate Limits
+
+From the [Bot FAQ](https://core.telegram.org/bots/faq#my-bot-is-hitting-limits-how-do-i-avoid-this), we know a few limits that cannot be exceeded, ever.
```

Comment:
```suggestion
From the [bot FAQ](https://core.telegram.org/bots/faq#my-bot-is-hitting-limits-how-do-i-avoid-this), we know a few limits that cannot be exceeded, ever.
```

Reply:
Inconsistent with how we refer to it in our FAQ. Inconsistent with how https://core.telegram.org/bots/api refers to it.

Reply:
If you want to go with Bot FAQ, you gotta remove the article.

Reply:
Why? I'm comparing it to a name, such as **the United States**.

Reply:
I had something in my train of thoughts, but it just crashed. I’m also confused. I don’t trust in how Telegram refers to it anyway. Maybe this needs a little more thinking by me.

Reply:
Telegram docs often make mistakes with articles, but even the top of https://core.telegram.org/bots/api can be read my way

Reply:
We can go with that now so the PQ is not blocked any further.